### PR TITLE
docs: made dark mode compatible APISIX architecture image

### PR DESCRIPTION
**Pull request by iamayushdas**
### What this PR does / why we need it:
To make the apisix architecture image compatible to dark mode
[#301](https://github.com/apache/apisix-website/issues/301) from apisix-website

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by iamayushdas** [comment]
@juzhiyuan @Yiyiyimu

**Review by juzhiyuan — APPROVED** [review]
Great!!!

File: docs/en/latest/config.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "version": 2.4,
   "sidebar": [
+    {
+      "type": "link",
+      "label": "Apache APISIX",
+      "href" : "https://apisix.apache.org/docs/apisix/getting-started"
```
Comment (spacewander) [inline docs/en/latest/config.json:7]: Why add this link?

**Review by spacewander — CHANGES_REQUESTED** [review]

File: docs/en/latest/config.json (hunk 1 above)
Comment (juzhiyuan) [reply docs/en/latest/config.json:7]: image

I see, @iamayushdas it seems that you have historical commits.

Comment (iamayushdas) [reply docs/en/latest/config.json:7]: Yes 😩 going to fix it

**Pull request closed without merging**